prestoadmin/util/presto_config.py

```diff
@@ -66,12 +66,11 @@ def from_file(config, config_path=None, config_host=None):
         return PrestoConfig(presto_config_dict, config_path, config_host)
 
     @staticmethod
-    def coordinator_config():
+    def config(host, user):
```

Comment:
1. Can you make the commit message more clear ("give inject host and user" + the description).  I know what this does, and still don't understand what the commit message is saying :) 

2.  This change doesn't seem to be used at all.  You have a different solution in the commit "Do not call fabric based function from product test".   I like this approach better because you can't accidentally call the wrong version.